flow/__init__.py

```diff
@@ -42,7 +43,10 @@
 ]
 
 
-if _get_config_value("import_packaged_environments", default=True):
+if _get_config_value(
```

Comment:
In light of our plans for moving more of the validation logic (hopefully including default handling) into signac's config API, let's leave this as is in this PR and revisit it later. Most aspects of our config handling feel like they need some work. I think the way that this should probably work is that signac-flow should wrap signac's `read_config` function to perform flow-specific validation so that we can put all default-handling into a standardized validate-with-defaults step, but I haven't thought that all the way through yet.